app/models/video.rb

```diff
@@ -0,0 +1,13 @@
+class Video < ApplicationRecord
```

Comment:
It would make sense to have a checkout_video method instead of doing that logic in the controller.